### ISSUE TITLE
"Fonts not loaded yet"

### ISSUE THREAD

**Issue by joydesign777** [photopea/photopea#6811]
Your Operating System: Windows 10
Your Web Browser: Google Chrome
The problem happens in the Incognito Mode, too: Yes, same problem

**This must be done to see the problem**
1. Go to www.photopea.com
2. Open previous file that have texts using local font on it
3. Ckick Type Tool and edit the text
4. When you try to edit the text, it said "Fonts not loaded yet"
5. Happens randomly

I have the Google Chrome permission and everything is on including "Font" option

**Comment by photopea** [comment]
Can you use non-local fonts in Photopea? Do they work well?

**Comment by joydesign777** [comment]
> Can you use non-local fonts in Photopea? Do they work well?

The same thing happens, I can't type or edit text using type tool when using local or non local font

I also tried on different PC, the same thing happens.

**Comment by joydesign777** [comment]
I tried to create new blank project, and the type tool doesn't work. I can't type anything

If I opened a saved project with text on it, I can't edit the text with error : "Fonts not loaded yet"

Tried this on my PC Win 10 Chrome, and Laptop Win 10 Chrome

**Comment by photopea** [comment]
Can you access our font files, e.g. this one? https://vecpea.com/rsrc/fonts/fs/LiterataBook-Medium.otf

**Comment by joydesign777** [comment]
> Can you access our font files, e.g. this one? https://vecpea.com/rsrc/fonts/fs/LiterataBook-Medium.otf

I can open the link you gave me if that what you meant by äccessing the font file

I saw that font already installed in Photopea

But the result is still the same. I can't type or edit any text

**Comment by joydesign777** [comment]
I also have tried using different Browser (Microsoft Edge) with the same result. Can't use type tools / font not loaded yet

**Comment by joydesign777** [comment]
> Can you download this OTF file to your computer? https://vecpea.com/rsrc/fonts/fs/LiterataBook-Medium.otf

Yes I can. I have downloaded and installed it in my PC. And I can use that font in Microsoft Excel/Word

**Comment by photopea** [comment]
Do you see any errors in a web console when the error happens?

**Comment by joydesign777** [comment]
I've stumbled on a solution. I tried to change my ISP using Hotspot tethering from my Phones, and it suddenly works again. Now I can use the Type tool....Strange

**Comment by photopea** [comment]
Can you open https://vecpea.com ? It should redirect you to Photopea. Are there any firewalls or censorship in your country?

**Comment by joydesign777** [comment]
> Can you open https://vecpea.com ? It should redirect you to Photopea. Are there any firewalls or censorship in your country?

Hi yes I can open the Photopea using vecpea link.

There are some firewall or censorship in my country depends on the ISP

The error regarding the Type tool / Font not loaded yet occurs randomly and I believe it's caused by the DNS from certain ISP in my country. Today I got no problem.

But if the problem would occur in the future, I will try one of these solutions :
1. Change the ISP
2. Use Cludflare WARP DNS
3. Try to open Photopea useing vecpea.com

Thank You
